Inflattices (#1319)
UniMath/agda-unimath: src/order-theory/lower-bounds-large-posets.lagda.md

  pr2 (logical-equivalence-is-binary-lower-bound-Π-Large-Poset z) =
    is-binary-lower-bound-is-binary-lower-bound-Π-Large-Poset z
```

> ### An element `x : Π-Large-Poset P` of a dependent product of large posets `P i` indexed by `i : I` is a lower bound of a family `a : J → Π-Large-Poset P` if and only if `x i` is a lower bound of the family `(j ↦ a j i) : J → P i` of elements of `P i`

Reviewer: This header is quite a mouthful.

I believe it is trying to say the following?

```suggestion
### An element of a dependent product of large posets is a lower bound if and only if it is a pointwise lower bound
```
